datashader/__init__.py

```diff
@@ -18,12 +18,35 @@
     from . import datatypes  # noqa (API import)
 
 # make pyct's example/data commands available if possible
-from functools import partial
+import warnings
+from functools import partial, wraps
+
+
+def _warn_pyct_deprecated(stacklevel=2):
+    warnings.warn(
+        "The 'pyct' package is deprecated since version 0.19 "
+        "and will be removed in version 0.20. For downloading sample datasets, "
+        "prefer using 'hvsampledata' (for example: "
+        "`hvsampledata.nyc_taxi_remote('pandas')`).",
+        category=FutureWarning,
+        stacklevel=stacklevel,
+    )
+
+
+def _deprecated_pyct_wrapper(func):
+    """Wrapper to add deprecation warning to pyct functions."""
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        _warn_pyct_deprecated(stacklevel=2)
```

Comment:
I'm not sure if these stack levels are set correctly, but they are likely still emitted because the category is a FutureWarning. 

You can test this out by setting the `CategoryWarning` to `DeprecationWarning`.

Reply:
You're right. Thanks. The correct stacklevel is 3 in this instance. I tested it via
```python
import datashader as ds

ds.fetch_data("datashader")
```